backend/src/routes/analytics.py

```
@@ -0,0 +1,92 @@
+    except Exception as e:
+        logger.error(f"Error fetching dashboard data: {str(e)}")
+        return error_response('Failed to fetch dashboard data', details=str(e))
+
+@analytics_bp.route('/transactions', methods=['GET'])
+async def get_transactions():
+    """Get recent transactions/trades"""
+    try:
+        limit = int(request.args.get('limit', 50))
+        trades = get_recent_trades(limit=limit)
+        return success_response(data=trades, count=len(trades))
+    except Exception as e:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)